pom.xml

```diff
@@ -38,12 +37,28 @@
     <dependencies>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy</artifactId>
+            <artifactId>quarkus-resteasy-reactive</artifactId>
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-openapi</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Why do we need all these?

Reply:
@tqvarnst can you update your PR and drop all the `jakarta.*-api` artifacts that were added? They were added by mistake and I just fixed the issue in the recipes (it was introduced relatively recently as a side effect of some other work).

Reply:
That was strange. Initially, when Clement asked about that, I was wondering what he was talking about, but apparently, the update tool added these dependencies. I've now update the PR and removed the jakarta dependencies.

Reply:
Yes, the Camel team also reported it to us. I fixed it a few minutes ago and the new recipes are on their way to Maven Central.